src/test/blk_non_zero/TEST0

```diff
@@ -43,11 +43,13 @@ require_test_type medium
 
 # doesn't make sense to run in local directory
 require_fs_type pmem non-pmem
+exclude_ppc64
```

Comment:
hmm, I think it would be helpful to have a comment which test for ppc64 matches which test for !ppc64